force-first-last.php

```diff
@@ -102,7 +134,7 @@ function ffl_fix_user_display_name( $user_id ) {
  *
  */
 function ffl_settings_menu_item() {
-	add_options_page('Force First Last', 'Force First Last', 'manage_options', 'ffl_settings', 'ffl_settings_page');
+	add_options_page('Force First Last', __( 'Force First Last', 'force-first-last' ), 'manage_options', 'ffl_settings', 'ffl_settings_page');
```

Comment:
Instead of adding an options page, could we add our option to the Settings -> General or Settings -> Discussion page?

Reply:
This has already been there but we could consider it in the future.